src/app/modules/gb-export-module/gb-export.component.ts

```diff
@@ -23,12 +23,11 @@ export class GbExportComponent implements OnInit, OnDestroy {
 
   private timer: Timer;
 
-  // TODO: fix export workflow:
   // 1. user defines cohort in cohort selection
-  // 2. user goes to export tab, variables on the left are selectable
-  // 3. based on cohort and variables, compose combination constraint to get data table
-  // 4. user arranges data table
-  // 5. user selects export data type, defines export job
+  // 2. user goes to export tab, the variables the left become selectable
```

Comment:
should be: "user goes to export tab, the variables on the left become selectable", will change